Chamasma/Chamasma/View/CategoryView.swift

```diff
@@ -39,15 +41,8 @@ public struct CategoryView: View {
 
     private var categoryItemList: some View {
         VStack(alignment: .leading) {
-            ForEach(1...10V, id: \.self) { _ in // TODO: will iterate given array
-                MealCell(
-                    name: "Cheesburger",
-                    description: "Cheesburger, Regular Bun, ketchup, pickle",
-                    price: 123.45,
-                    rating: 0.78,
-                    image: DesignSystem.Image.burger,
-                    isPopular: true
-                )
+            ForEach(meals, id: \.self) { meal in
```

Comment:
Use if instead of self. 

If you make meal identifiable, then no `id: ...` is needed